release-notes/Cube/Cube_Feature_Release_10.5/Cube_10.5.6.md

```diff
@@ -62,6 +62,9 @@ In the *Augmented Operations alarm settings* window, which you can access in the
 
 If already installed on your system, the app will open. If not, you will be referred to the [RAD Manager page in the Catalog](https://catalog.dataminer.services/details/174b9848-43c8-470d-afc2-1b1722f05e74).
 
+> [!NOTE]
+> On the Main Release track, this link is only available from DataMiner 10.6.0 onwards.
```

Comment:
@tobedeprez, OK if we adjust this like this, in case people are using the latest Cube version with an older DataMiner server version? 

```suggestion
> This link will only be available if the DataMiner server uses DataMiner 10.5.4/10.6.0 or higher.
```